packages/framework/ai-collab/src/aiCollabApi.ts

```diff
@@ -182,3 +186,76 @@ export interface TokenLimits {
 	 */
 	readonly outputTokens?: number;
 }
+
+/**
+ * Represents a path through a tree of objects.
+ * number values represent array indices whereas string values represent object keys.
+ *
+ * @alpha
+ */
+export type ObjectPath = (string | number)[];
+
+/**
+ * Represents a create operation between two branches of a tree.
+ * Meaning that an attribute (a shared tree node) was identified as being created.
+ *
+ * @alpha
+ */
+export interface DifferenceCreate {
```

Comment:
I think it would be better to call this `DifferenceInsert` since it's an array only operation.

Reply:
Updated